nixos/modules/config/shells-environment.nix

```diff
@@ -169,7 +169,7 @@ in
          ${cfg.extraInit}
 
          # The setuid wrappers override other bin directories.
```

Comment:
I do think that's a good idea, comment-debt is as bad as tech-debt. Thank you for pointing this out.